Snowplow/SPDevicePlatform.h

```diff
@@ -0,0 +1,20 @@
+//
+//  SPDevicePlatform.h
+//  Snowplow
+//
+//  Created by Alex Benini on 18/12/2019.
+//  Copyright © 2019 Snowplow Analytics. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+typedef NSString * const SPDevicePlatform NS_TYPED_EXTENSIBLE_ENUM;
```

Comment:
Would this be better as a NS_TYPED_ENUM? 
The Snowplow Tracker Protocol doesn't support anything beyond the ones listed and events end up erroring with a custom platform:
"Field [p]: [newPlatform] is not a supported tracking platform"

Reply:
Oh right, I thought the collector was already able to manage custom entries but it still an open issue in the snowplow repo.

Reply:
Looks like this is still open https://github.com/snowplow/snowplow/issues/2814 
I'd suggest once this is done then we update the trackers to support custom platforms but until then, we just run the risk of users creating bad events.